### ISSUE TITLE
Confusing button

### ISSUE THREAD

**Issue by acetrand** [hengun348/GitHub-compare-helper#1]
The button "Generate..." unexpectedly copies the URL to my clipboard. I much rather want to decide for myself if I want to copy it or not. Eventually I probably will copy it but first I would like to see that I get the right commits. Especially as the "range-picking" method is not that obvious.

Couldn't the button just navigate to the compare-page? The copy-to-clipboard feature could be as github have done it, with a separate icon.

![image](https://cloud.githubusercontent.com/assets/4696698/15468059/42c74dfe-20e2-11e6-8ca1-cd2fdb1acc1b.png)

**Comment by hengun348** [comment]
Thank you for your input @acetrand. 

I first thought that it was best to directly copy the compare url to the clipboard when you press "Generate compare url" that is why I implemented it that way. But since I have been using this plugin for a while I have started to agree with you, I also would like to check that I actually got the right commits before the url is copied.

I will add this to my todo list for possible improvements. I think I prefer this to be an option then so the user can decide for themselves how they want it.
